src/containers/shared/components/TransactionTable/TransactionTable.tsx

```diff
@@ -36,6 +42,7 @@ export const TransactionTable: TransactionTableComponent = ({
     <>
       <ol className="transaction-table">
         <li className="transaction-li transaction-li-header">
+          {hasTokensColumn && <div className="col-token"> {t('token')} </div>}
```

Comment:
I really don't like the idea of leaving an empty Tokens column. Can this part be moved to a different PR? You can just comment out this code if you don't want to remove the other parts, I just don't want it visible.

Reply:
The column has an option to be hidden - I just hid it and I'll unhide it in the next PR.